### PR TITLE
Resolved some minors issue with Suite8 wiki content

### PR DESCRIPTION
**Pull request by Mac-Rae**

### CODE REVIEW

File: content/8.x/admin/Compatibility Matrix.adoc [hunk 1]
```diff
@@ -16,14 +16,11 @@ The following documentation is for SuiteCRM Version 8+; to see documentation on
 
 2+^h| Platform 
 
-| Linux, Unix, Mac OS | Any version supporting PHP 
-
-| Windows | Windows Server 2012+
-
+| Linux, Unix, Mac OS | Any version supporting PHP
 | PHP | 7.3, 7.4, 8.0
 | Angular CLI | 11+
```
Comment (mattlorimer) [inline content/8.x/admin/Compatibility Matrix.adoc:21]: Angular, Node, yarn are all dev dependencies, may put them in separate section

**Review by mattlorimer — CHANGES_REQUESTED** [review]

File: content/8.x/admin/Compatibility Matrix.adoc (hunk 1 above)
Comment (mattlorimer) [reply content/8.x/admin/Compatibility Matrix.adoc:21]: all atomic commit if possible

Comment (Mac-Rae) [reply content/8.x/admin/Compatibility Matrix.adoc:21]: Whoops, though i had actually already done that, will update.
And will rebases to fix the commit issue

Comment (Mac-Rae) [reply content/8.x/admin/Compatibility Matrix.adoc:24]: Should be good for review

**Comment by Mac-Rae** [comment]
@mattlorimer reviewed by craig, can you code review and hopefully merge? :laughing:

**Review by mattlorimer — APPROVED** [review]

**Comment by Mac-Rae** [comment]
Whoop whoop

**Pull request merged**